data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset

```diff
@@ -15,7 +15,7 @@ local Node = {
     File = asset.resource("data/dummydata.csv"),
     -- Change the orientation render option to face the camera position instead
     -- of its view direction
-    OrientationRenderOption = "Camera Position Normal",
+    Billboard = "Camera Position Normal",
```

Comment:
OBS breaking change

And the docs for the point clouds have to be updated: https://docs.openspaceproject.com/releases-v0.20/content/point-data/point-data.html#orientation